src/script/media/MediaStreamHandler.ts

```diff
@@ -143,11 +143,11 @@ export class MediaStreamHandler {
     isGroup: boolean,
     hasPermission: boolean,
   ): Promise<MediaStream> {
-    const mediaContraints = screen
+    const mediaConstraints = screen
       ? this.constraintsHandler.getScreenStreamConstraints(this.screensharingMethod)
       : this.constraintsHandler.getMediaStreamConstraints(audio, video, isGroup);
 
-    this.logger.info(`Requesting MediaStream`, mediaContraints);
+    this.logger.info(`Requesting MediaStream`, mediaConstraints);
```

Comment:
```suggestion
    this.logger.info('Requesting MediaStream', mediaConstraints);
```

Reply:
I leave this up for the linter to decide.